pages/meetings/office_hours.md

```diff
@@ -7,6 +7,8 @@ hidetitle: True
 
 # PlasmaPy "Office" Hours
 
+**Note: We will not be holding office hours on 2020 December 24 or 2020 December 31, but will return to our regularly scheduled programming on 2021 January 7th.**
```

Comment:
```suggestion
<p style="color: var(--plasmapy-red)"><b>Note: We will not be holding office hours on 2020 December 24 or 2020 December 31, but will return to our regularly scheduled programming on 2021 January 7th.</b></p>
```

This will highlight the note in the PlasmaPy red.